backend/src/apiserver/archive/log.go

```diff
@@ -294,15 +296,14 @@ func (a *LogArchive) CopyLogFromArchive(logContent []byte, dst io.Writer, opts E
 	return nil
 }
 
-func decompressLogArchive(logContent []byte) (reader io.Reader, err error) {
+func decompressLogArchive(logContent []byte) (io.Reader, error) {
```

Comment:
Maybe I'm missing something, but is there any specific reason to change the return format here? Same for several functions below.

Reply:
Explicit return arguments are recommended for better readability and smaller chances of introducing bugs.